commons/src/store/mailbox.ts

```diff
@@ -228,6 +228,25 @@ function initializeThreads() {
             }
             return { ...threads };
           });
+        } else {
+          // This happens when someone sends a message using composer and we want
+          // to update the existing thread with the sent message
```

Comment:
Is this the only case in which we'll hit this `else`?

Reply:
That's correct. Apart from this change, `hydrateMessageInThread` is called when the messages array has at least 1 message.